### PR TITLE
Refactor storybook examples

### PR DESCRIPTION
**Pull request by ryanjduffy**
* Adds `shared/storybook` module to share selectors and utilities for storybook-based examples
* Refactors the two storybook examples to use the example helper and the shared module

https://replay.io/view?id=81b3b873-3b45-41f7-96c0-ffdcf5af38b3
https://replay.io/view?id=ea06084f-1f6b-468a-95e7-77299e8633a3

### CODE REVIEW

File: examples/shared/storybook.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    return `[data-item-id="${id}"]`;
+  },
+};
+
+// Navigates to either a story or a folder (if the path includes a trailing slash).
+// A leading slash is optional.
+const openStory = async (action, path, callback) => {
+  await action(`Open story: ${path}`, async (page, ...rest) => {
+    // await Promise.all([
+    // page.waitForNavigation(),
```
Comment (jasonLaster) [inline examples/shared/storybook.js:28]: I assume we dont need this waitForNavigation

File: examples/shared/storybook.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const selectors = {
+  toolbarButtonByTitle: function (title) {
+    return `button[title="${title}"]`;
+  },
+  tabByTitle: function (title) {
+    return `button[role="tab"]:text("${title}")`;
```
Comment (jasonLaster) [inline examples/shared/storybook.js:6]: `:text()` could be really powerful

**Review by jasonLaster — APPROVED** [review]
Looks great!

File: examples/shared/storybook.js (hunk 1 above)
Comment (ryanjduffy) [reply examples/shared/storybook.js]: Nope. I cleaned this up locally but missed it in my commit. `waitForNavigation` seems to only be required for full-page reloads (and then maybe only when it's not triggered by click/form submit). Cleaned it up!

**Pull request merged**